configuration/persistence.md

```diff
@@ -11,59 +11,73 @@ openHAB can store data over time; this is known as persistence.
 The data may be retrieved at a later time, for example to restore your system after startup, or to prepare graphs for display on a UI.
 
 openHAB persists Item states in a database, and most popular databases are supported.
-You may have more that one persistence add-on loaded, and each of these may be configured independently.
+You may have more than one persistence add-on loaded, and each of these may be configured independently.
 
 A complete list of supported persistence add-ons may be found in the [persistence]({{base}}/addons/persistence.html) section of the on-line openHAB documentation.
 
-## Persistence Add-on Configuration
+## Add-on Configuration
 
-Each persistence add-on you install will need to be configured.
 Please refer to the specific [on-line documentation]({{base}}/addons/persistence.html) for your selected persistence add-on for configuration instructions.
 
 ## Default Persistence Service
 
-You may install more than one persistence add-on.
-Therefore, it is important to select a default persistence service.
+It is important to select a default persistence service.
 You should do this even if you have only one persistence add-on installed.
 
 To select a default persistence service, in paper UI, select Configuration and then System from the side menu.
-Scroll down to "Persistence", and select your Default Service from the drop-down list.
+Scroll down to "Persistence", and select your default service from the drop-down list.
 Note that you must first install a persistence add-on before you make this selection.
 Be sure to save your choice once you have selected your default service.
 
-## Configuration
+## Persistence Configuration
 
-Persistence Strategies are configured in a file named `<persistenceservice>.persist`, stored in `$OPENHAB_CONF/persistence` ("persistenceservice" is replaced by the name of your add-on (e.g. `rrd4j.persist`)).
+Persistence Strategies are configured in a file named `<persistenceservice>.persist`, stored in `$OPENHAB_CONF/persistence` (Replace "persistenceservice" with the name of your persistence add-on (e.g. `rrd4j.persist`)).
```

Comment:
double brackets? 😕

Reply:
How about one sentence to introduce the concept. Short example: "Configuration defines which Item, when, where."

Reply:
Agreed.